src/ffbwrapper.c

```diff
@@ -192,11 +216,14 @@ static void init()
             exit(-1);
         }
 
-        timerspec.it_value.tv_sec = 0;
-        timerspec.it_value.tv_nsec = 3e6;
-        timerspec.it_interval.tv_sec = 0;
-        timerspec.it_interval.tv_nsec = 3e6;
-        timer_settime(throttle_timer_id, 0, &timerspec, NULL);
+        get_timer_interval(&timerspec.it_interval);
+        timerspec.it_value = timerspec.it_interval;
+        report("# FFBTOOLS_THROTTLING_INTERVAL_NS=%lu", timerspec.it_interval.tv_nsec);
```

Comment:
Sounds good. I didn't merge it with that line initially in case the `FFBTOOLS_THROTTLING` variable wasn't set, but now that we're combining those, I'll update that.